damn-fast-priority-queue/test.lisp

```diff
@@ -12,15 +12,16 @@
 ;;;; Utilities
 
 (defun verify-heap-property (vector)
+  (declare (type q::prio-vector-type vector))
   (loop with length = (length vector)
         for parent from 0 below (truncate length 2)
         for left = (+ (* parent 2) 1)
         for right = (+ (* parent 2) 2)
-        do (assert (< (aref vector parent) (aref vector left)) ()
+        do (assert (<= (aref vector parent) (aref vector left)) ()
```

Comment:
In which case can these two be equal? Objects with equal priorities, or something?

Reply:
yeah, judging by the commit comment and the added test, that seems to be the intent.